cp22/hpi-fun2/hexhex/executables/solution.py

```diff
@@ -0,0 +1,50 @@
+import io
+import os
+
+from regex import X
```

Comment:
Remove this import, it results in a run error as the `regex` package is not available on the judge.